api/templates/error_template.txt

```diff
@@ -0,0 +1,9 @@
+S'ha rebut un error de KOIKI per a la comanda: {{url_wc_order}}
```

Comment:
I'd try to make as clear as possible from the perspective of customer support. What about

```
La notificació a l'empresa d'enviament (Koiki) de la comanda {{url_wc_order}} ha fallat amb el següent error:

{{error_returned}}

La comanda romandrà en estat "en espera <replace with the right state>" i no serà enviada al client fins que no ...
```

Also, are we sure `self.req_body` contains only the creation date? won't it be more confusing than anything to output the entire body? Perhaps we could print them as `detalls de l'enviament`?

Reply:
What do you mean by the "creation date" ? Sorry i think there is a missunderstanding with "Data enviada", i meant Dades enviades... but I will change it to "Detalls de l'enviament"...

Currently the req_body contains I believe necessary debug information (see screenshot [here](https://github.com/coopdevs/lazona_connector/issues/82#issuecomment-866772078) ) 

To find out what is Koiki receiving in case we need to see if an error is comming because of a missuse of a single or doble quotes, etc... 

If admins find it too confusing I guess it can be shown as pretty format or remove some fields.

Reply:
This email needs to be actionable by customer support so they can act on errors, if possible. So to do that, I would definitely make the email contents very precise and readable. That is, pretty-printing the dictionary as key-value pairs and using the appropriate headers like `Detalls de l'enviament`.

The least we want to deal with is errors on the error tracker and customer support people misunderstanding these emails and taking too much of our time.

Reply:
email prettyfied: 
![image](https://user-images.githubusercontent.com/56868560/123618393-72a9ca00-d808-11eb-8971-4ea203effc44.png)